### DISCUSSION TITLE
如果你有想法或者建议，可以在这里发布。

### DISCUSSION THREAD

**Discussion by KuiyueRO** [KuiyueRO/Obsidian-Miner#7] [General]
如果你有发现bug，那么请在issues中发布 : P

**Comment by randeng798** [comment]
大佬，不知道为什么我按照你的步骤做了之后，小logo的标题背景是黑色的，并没有底下标题出现
![1](https://github.com/user-attachments/assets/9b3bb28e-4fbf-46a5-a93c-3bbd2dd53779)

**Reply by KuiyueRO** [reply]
你的cssclasses是怎么设置的

**Reply by KuiyueRO** [reply]
还有你的banner和图标源码是什么样的
